### DISCUSSION TITLE
Allow for linking cross-space, with version

### DISCUSSION THREAD

**Discussion by rachellawson** [GitbookIO/community#38] [Feature Requests]
### What's your feature/idea?

For a number of reasons, we have some documentation spread across a few spaces. We would like to be able to interactively add page links to the other spaces AND specify the version to which the ink should apply.

### What's your desired outcome?

I would see a new paragraph type, “Inter-space page link”, that asked three questions:

1. Which space to connect to
2. Which version within that space to connect to
3. Which page to connect to
And then maintain that connection.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by rodrcastro** [comment]
Hey @rachellawson! Thank you for your suggestion here. We really appreciate you using the community to share this :)

If I understood your request correctly, you'd like to create a link inside a page to a page in another space.

We have a way for you to do that. Here's a short video showing it in action:

https://user-images.githubusercontent.com/18582098/215845314-72499edb-efd4-45a0-8bb7-10df58e8137a.mp4

You can use either the inline palette (we show more of the palette in our [changelog here](https://changelog.gitbook.com/december-2022/15-december-22-editor-improvements-and-more#inline-palette-in-the-editor)) or you can use the @ symbol as shown in the video.

Let me know if that was what you were looking for!

**Comment by rachellawson** [comment]
Ah - that is awesome BUT not quite what I need - as we have multiple versions of each space, they are all in their own collection. So, there is a named collection (see attached screenshot) and then a Development space for each, plus future versioned spaces. 

In this example, I want to link from the Development space in the GovStack Specification collection to the Development space in the bb-information-mediation collection. That wouldn't appear in the @-link tool mentioned above.

<img width="1552" alt="Screenshot 2023-02-01 at 16 37 25" src="https://user-images.githubusercontent.com/807248/216105694-e951134d-e04c-406b-bbc6-508fb33702f0.png">

I believe I'm describing new functionality that will be useful to many users, especially the bigger ones.

(Oh, and whenever listing spaces in dialogs, you might want to include the collection name, too. I often see a long list of "Development, Development, Development... 😉)
